src/resources/app/pages/course/Announcement.vue

```diff
@@ -11,6 +12,9 @@ import AppLayout from "../../components/Layout/AppLayout";
 export default {
   layout: [AppLayout, CourseStream],
   components: {AnnouncementContainer},
+  props:{
+    id:{},
```

Comment:
please add one space before the brackets

Reply:
and why the default value of id is an object?

Reply:
My bad, should be number, am I right?

Reply:
yes, or you can just put `null`. Either one is okay

Reply:
> yes, or you can just put `null`. Either one is okay

never mind, I think it can't be null lol